x-pack/plugins/observability_solution/apm/server/routes/errors/get_error_groups/get_error_group_sample_ids.ts

```diff
@@ -42,6 +43,7 @@ export async function getErrorGroupSampleIds({
   start: number;
   end: number;
 }): Promise<ErrorGroupSampleIdsResponse> {
+  const requiredFields = asMutableArray([ERROR_ID] as const);
```

Comment:
Do we need to adapt this? https://github.com/jennypavlova/kibana/pull/9#discussion_r1787844150
CC @dgieselaar @MiriamAparicio

Reply:
No idea. I'll wait for Dario to chime in.

Reply:
no because we only use `error.id` in this function